docs/download.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: page
+title: Download
+nav_order: 10
```

Comment:
I suggest making this a much lower nav order, meaning it's much higher on the list and therefore easier to see.  Placing it last makes it look like we're trying to hide it and it's an afterthought.  Just after or before the "Getting Started" section seems like a logical order to me.